Automattic/sensei
# Add Onboarding Wizard page

#### Pending questions:

* **Fullscreen Onboarding Wizard:** 
   Does this page need to be fullscreen?
   Woo has a component that supports this, but it is not really exposed to be as an externally usable component right now. It works by hiding all the menus with CSS, which we can copy. 
* **Dismissing**:
   Can the Onboarding Wizard be closed any time without going through?  What pages do we want to redirect to it until the user finished/dismissed it?

* **Redirect to Wizard after plugin activation:**
  Added a new card for this as it is not trivial, and also partially depends on the previous question. 

#### Changes proposed in this Pull Request:

* Add Onboarding class
* Add a Sensei submenu page for onboarding. The menu item should be hidden later, but now it makes testing easier.
* Add JSX file that renders the React-based on that page
* Add components from Wordpress and Woocommerce component libraries, with styles

#### Testing instructions:

* Requires `change/build-webpack` branch
  * Merge that branch into this
  * Run `npm install`
  * Add `'onboarding/onboarding.jsx'` to `webpack.config.js`
  * Run `npm start` or `npm run build` 
* Open page in wp-admin under Sensei LMS -> Onboarding
* Page should have a card, heading and a button, properly styled.
